Geometry/CSCGeometryBuilder/src/CSCGeometryParsFromDD.cc

```diff
@@ -426,24 +453,31 @@ bool CSCGeometryParsFromDD::build(const cms::DDCompactView* cview,
       }
 
       auto wirespacing = fv.get<double>("WireSpacing");
-      wg.wireSpacing = static_cast<double>(wirespacing);
+      wg.wireSpacing = static_cast<double>(wirespacing / dd4hep::cm);
+      edm::LogVerbatim("CSCGeometryParsFromDD") << "(2) wireSpacing: " << wg.wireSpacing / dd4hep::cm;
 
       auto alignmentpintofirstwire = fv.get<double>("AlignmentPinToFirstWire");
-      wg.alignmentPinToFirstWire = static_cast<double>(alignmentpintofirstwire);
+      wg.alignmentPinToFirstWire = static_cast<double>(alignmentpintofirstwire / dd4hep::cm);
+      edm::LogVerbatim("CSCGeometryParsFromDD")
+          << "(3) alignmentPinToFirstWire: " << wg.alignmentPinToFirstWire / dd4hep::cm;
```

Comment:
I didn't notice this before. `wg.alignmentPinToFirstWire` has already been converted to cm. It doesn't need a second conversion to cm. Same applies to the other debug print statements below.

Reply:
@cvuosalo : I added "/dd4hep::cm" in order to put the units in the code for a future fix even if alignmentPinToFirstWire is already in cm. Adding "/dd4hep::cm" do not change anything if the value of the parameter is already in "cm"